CarCrash.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ */
+
+package com.mycompany.carcrash;
+
+import com.mycompany.carcrash.CustomExceptions.BrakeFailException;
+import com.mycompany.carcrash.CustomExceptions.CarAccidentException;
+import com.mycompany.carcrash.CustomExceptions.EngineNotWorkingException;
+
+/**
+ *
+ * @author ria.mishra
+ */
+public class CarCrash {
+
+    public static void main(String[] args) {
+        Car car = new Car();
+        EmergencyService emergencyService = new EmergencyService();
+        try
+        {
+            car.driveCar();
+        }
+        catch(BrakeFailException ex)
+        {
+            emergencyService.reportToAuthorities();
+            emergencyService.initateEngineBreak();
+        }
+        catch(CarAccidentException ex)
+        {
+            emergencyService.openAirBags();
+            emergencyService.checkForInjuries();
+            emergencyService.reportToAuthorities();
+            emergencyService.notifyInsuranceCompany();
+            emergencyService.towCarToServiceCenter();
+        }
+        catch(EngineNotWorkingException ex)
```

Comment:
We can have good name for the exception.

Reply:
Done the required changes.